doc/_ext/nope/scopes/histogram.py

```diff
@@ -0,0 +1,12 @@
+from pynopegl_utils.misc import SceneCfg, scene
+
+import pynopegl as ngl
+
+
+@scene()
+def histogram(cfg: SceneCfg):
+    jpegs = (m for m in cfg.medias if m.filename.endswith(".jpg"))
```

Comment:
images + (jpg or png)

Reply:
Same as previous PR, I'm looking for a rich picture (to have significant histogram and waveforms)